Resources/Textures/Objects/Consumable/Drinks/david_martinez.rsi/meta.json

```diff
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi", "states": [{"name": "icon"}]}
```

Comment:
rsi should be renamed to the_martinez.rsi

Reply:
Im not at my home PC rn.  Ill update it when i get home in a few hours.